openedx/features/caliper_tracking/tests/current/edx.course.enrollment.activate.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
The name of the event is `edx.course.enrollment.activated`

not `edx.course.enrollment.activate`

Reply:
fixed